docs/contributing/CHANGELOG_TEMPLATE.md

```diff
@@ -4,26 +4,21 @@
 
 💥 Breaking changes:
 
-- Pull Request Title goes here
+Heading (h4) to summarise change goes here
 
-  Description goes here (optional)
+Description goes here
```

Comment:
@EoinShaughnessy Should we give some more information on what to include in the description for a breaking change? Such as telling the user what change they need to make and in what circumstances (i.e: if it only affects people using the HTML)?

Reply:
Sounds good!

```suggestion
Description goes here - tell the user what change they need to make, and in what circumstances. For example, if the change only affects users who are using HTML.
```

Reply:
Wonder if we should follow what we did for v4.0.0 and refer to users who are "not using Nunjucks macros" rather than "users who are using HTML"?

Reply:
That sounds really good, yeah let's do that:

```suggestion
Description goes here - tell the user what change they need to make, and in what circumstances. For example, if the change only affects users who are not using Nunjucks macros.
```